docs/sql-data-sources-json.md

```diff
@@ -155,6 +155,12 @@ Data source options of JSON can be set via:
     <td>Allows leading zeros in numbers (e.g. 00012). If None is set, it uses the default value, <code>false</code>.</td>
     <td>read</td>
   </tr>
+  <tr>
+    <td><code>inferDateType</code></td>
+    <td>false</td>
+    <td>Infers all DateType format for the JSON. If this is not set, it uses the default value, <code>false</code>.</td>
```

Comment:
**the** JSON ? Is it needed?

Reply:
this is to specify for the JSON in this doc